config.m4

```diff
@@ -6,6 +6,8 @@ if test "$PHP_UUID" != "no"; then
   for dir in $PHP_UUID /usr/local /usr; do
     if test -f "$dir/include/uuid/uuid.h"; then
       PHP_UUID_DIR="$dir"
+    elif test -f "$dir/include/ossp/uuid.h"; then
+      PHP_UUID_DIR="$dir"
```

Comment:
probably simpler to define UUID_ON_MAC here (rather than using AC_TRY_COMPILE)

Reply:
There is no guarantee that this is on mac though.

Reply:
AFAIK anyone can install http://www.ossp.org/pkg/lib/uuid/

Reply:
But, if this is a lib, you need to add them in builder options.....